app/preference.go

```diff
@@ -10,41 +10,42 @@ import (
 )
 
 func (a *App) GetPreferencesForUser(userId string) (model.Preferences, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().GetAll(userId); result.Err != nil {
+	result := <-a.Srv.Store.Preference().GetAll(userId)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else {
-		return result.Data.(model.Preferences), nil
 	}
+	return result.Data.(model.Preferences), nil
 }
 
 func (a *App) GetPreferenceByCategoryForUser(userId string, category string) (model.Preferences, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().GetCategory(userId, category); result.Err != nil {
+	result := <-a.Srv.Store.Preference().GetCategory(userId, category)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else if len(result.Data.(model.Preferences)) == 0 {
+	}
+	if len(result.Data.(model.Preferences)) == 0 {
 		err := model.NewAppError("getPreferenceCategory", "api.preference.preferences_category.get.app_error", nil, "", http.StatusNotFound)
 		return nil, err
-	} else {
-		return result.Data.(model.Preferences), nil
 	}
+	return result.Data.(model.Preferences), nil
 }
 
 func (a *App) GetPreferenceByCategoryAndNameForUser(userId string, category string, preferenceName string) (*model.Preference, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().Get(userId, category, preferenceName); result.Err != nil {
+	result := <-a.Srv.Store.Preference().Get(userId, category, preferenceName)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else {
-		data := result.Data.(model.Preference)
-		return &data, nil
 	}
+	data := result.Data.(model.Preference)
+	return &data, nil
 }
 
 func (a *App) UpdatePreferences(userId string, preferences model.Preferences) *model.AppError {
 	for _, preference := range preferences {
 		if userId != preference.UserId {
 			return model.NewAppError("savePreferences", "api.preference.update_preferences.set.app_error", nil,
-				"userId="+userId+", preference.UserId="+preference.UserId, http.StatusForbidden)
+				"userId="+userId+", preference.userId="+preference.UserId, http.StatusForbidden)
```

Comment:
Please try to avoid making cosmetic changes to code you're not otherwise changing since it can conflict with other PRs, and it can make it difficult to read `git blame` when trying to understand older changes

Reply:
Okay, fixes this.